grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -153,6 +153,17 @@ def section_compaction(outer_panels):
                         ),
                     ],
                 ),
+                panels.timeseries_bytesps(
+                    "Commit Flush Bytes(MB/s) by Table",
+                    "The  of bytes that have been written by commit epoch per second.",
+                    [
+                        panels.target(
+                            f"sum(rate({metric('storage_commit_write_throughput')})) by (table_id)",
```

Comment:
`storage_commit_write_throughput` is reporting `Bytes` instead of `MB`, we need to do a conversion